src/app/view2/view2.component.scss

```diff
@@ -1,7 +1,18 @@
-p {
+.container {
   width: 100%;
   height: 100%;
   background-color: aliceblue;
   margin: 0;
   padding: 10px;
 }
+
+.block {
+  width: 100px;
+  height: 40px;
+  background-color: blueviolet;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.